djstripe/models/payment_methods.py

```diff
@@ -1,8 +1,10 @@
+import sys
 from typing import Optional, Union
 
 import stripe
 from django.db import models, transaction
 from stripe.error import InvalidRequestError
+from stripe.six import reraise
```

Comment:
Any reason you're using six here instead of native reraising?

Reply:
Outdated